Source/Core/Core/AchievementManager.h

```diff
@@ -16,6 +17,10 @@
 #include "Common/Event.h"
 #include "Common/WorkQueueThread.h"
 
+using AchievementId = u32;
+static constexpr int RP_SIZE = 65536;
```

Comment:
```suggestion
constexpr int RP_SIZE = 65536;
```

constants are already internally linked by default in C++